lib_features.php

```diff
@@ -110,8 +115,13 @@ function send_course_request_mail($recipient_username, $course, $request_id) {
     $params->b = $CFG->wwwroot.'/user/view.php?id='.$USER->id;
     $params->c = utf8_encode($course->titel);
     $params->d = $CFG->wwwroot.'/local/lsf_unification/request.php?answer=12&requestid='.$request_id;
-    $content = get_string('email2','local_lsf_unification',$params);
-    return email_to_user($user,  get_string('email_from','local_lsf_unification')." (by ".$USER->firstname." ".$USER->lastname.")", get_string('email2_title','local_lsf_unification'),$content);
+
+    $jsondata = json_encode(array('userid' => $user->id, 'userfirstname' => $USER->firstname,
```

Comment:
There are still occurrences of `json_encode()`.

Reply:
Sorry, finally fixed with 990afac5ef245f42f937f04f4b3ebabf8c4294de.